src/common/catalog/Cargo.toml

```diff
@@ -17,5 +17,5 @@ snafu = { version = "0.7", features = ["backtraces"] }
 
 [dev-dependencies]
 chrono.workspace = true
-tempdir = "0.3"
+tempfile = "3.4.0"
```

Comment:
Can we move the `tempfile` to the project root `cargo.toml`?And use `tempfile.workspace = true` for these dependencies in every crate.
Looks like the `tempfile` is required in many crates. Using workspace dependency can simplify management.

Reply:
@killme2008 Could you please take a look at changes that address your suggestion when you have time?